src/dotnet-scaffolding/dotnet-scaffold-aspnet/Extensions/EfControllerScaffolderBuilderExtensions.cs

```diff
@@ -108,4 +111,81 @@ codeModifierProperties is not null &&
 
         return builder;
     }
+
+    public static IScaffoldBuilder WithMvcViewsStep(this IScaffoldBuilder builder)
+    {
+        builder = builder
+            .WithStep<ValidateViewsStep>(config =>
+            {
+                var step = config.Step;
+                var context = config.Context;
+                var addViews = context.GetOptionResult<bool>(Constants.CliOptions.ViewsOption);
+                if (!addViews)
+                {
+                    step.SkipStep = true;
+                    return;
+                }
+
+                step.Project = context.GetOptionResult<string>(Constants.CliOptions.ProjectCliOption);
+                step.Model = context.GetOptionResult<string>(Constants.CliOptions.ModelCliOption);
+                step.Page = BlazorCrudHelper.CrudPageType;
+            })
+            .WithStep<TextTemplatingStep>(config =>
+            {
+                var step = config.Step;
+                var context = config.Context;
+                var addViews = context.GetOptionResult<bool>(Constants.CliOptions.ViewsOption);
+                if (!addViews)
+                {
+                    step.SkipStep = true;
+                    return;
+                }
+
+                context.Properties.TryGetValue(nameof(ViewModel), out var viewModelObj);
+                ViewModel viewModel = viewModelObj as ViewModel ??
+                    throw new InvalidOperationException("missing 'ViewModel' in 'ScaffolderContext.Properties'");
```

Comment:
Adding the name of the `option` (e.g., `ViewsOption`) to the exception message would make it easier to spot what step caused the issue.

Without looking at the stack trace, this exception and the one from L.174 will have the very same error message.

Reply:
sure yes sg